robot/Cumulus/tests/browser/relationships/create_relationships.robot

```diff
@@ -1,48 +1,62 @@
 *** Settings ***
 
 Resource        robot/Cumulus/resources/NPSP.robot
-Suite Setup     Open Test Browser
+Library         cumulusci.robotframework.PageObjects
+...             robot/Cumulus/resources/ContactPageObject.py
+...             robot/Cumulus/resources/AccountPageObject.py
+...             robot/Cumulus/resources/RelationshipPageObject.py
+...             robot/Cumulus/resources/NPSP.py
+Suite Setup     Run keywords
+...             Open Test Browser
+...             Setup Test Data
 Suite Teardown  Delete Records and Close Browser
-Test Teardown   Capture Screenshot on Failure
+
+***Keywords***
+
+Capture Screenshot on Failure
+    Run Keyword If Test Failed  Capture Page Screenshot
+
+Setup Test Data
+    &{contact1} =                        API Create Contact    Email=automation@example.com
+    Store Session Record                 Account               &{contact1}[AccountId]
+    Set suite variable                   &{contact1}
+    &{contact2} =                        API Create Contact
+    Store Session Record                 Account               &{contact2}[AccountId]
+    Set suite variable                   &{contact2}
 
 *** Test Cases ***
 
 Create Relationships for contacts
-    [tags]  unstable
-    &{contact1} =  API Create Contact    Email=skristem@robot.com
-    Store Session Record    Account    &{contact1}[AccountId]
-    Go To Record Home  &{contact1}[AccountId]
+    [Documentation]                      Create 2 contacts with API which inturn creates 2 household accounts.
+    ...                                  Open contact2 record and create a new relationship with contact 1.
+    ...                                  Verify the parent-child relationship is correctly established.
+
+    [tags]                               W-037650                                                      feature:Relationships
```

Comment:
Personally I find this test hard to read when the keyword arguments are way over to the right. It's not a showstopper, but I recommend moving some of these arguments to the left.